src/GRA.Controllers/ProfileController.cs

```diff
@@ -1175,8 +1218,8 @@ public async Task<IActionResult> History(int page = 1)
                             bundle.AvatarItems.First().Thumbnail);
                         if (bundle.AvatarItems.Count > 1)
                         {
-                            var bundleLink = Url.Action(nameof(AvatarController.Index), 
-                                AvatarController.Name, 
+                            var bundleLink = Url.Action(nameof(AvatarController.Index),
```

Comment:
Hmm, should this be in this commit?

Reply:
Unsure of why I made this.... Will put back!